pgjdbc/src/test/java/org/postgresql/test/jdbc2/BatchExecuteTest.java

```diff
@@ -1383,4 +1384,51 @@ public void testNoServerPrepareOneRow() throws SQLException {
       TestUtil.closeQuietly(ps);
     }
   }
+
+  @Test
+  public void testMerge() throws Exception {
+    if (!TestUtil.haveMinimumServerVersion(con, ServerVersion.v15)) {
+      return;
+    }
```

Comment:
For reference, this should be `assume` so the test is displayed as `skipped` rather than `successful`.